src/main/java/ch/njol/skript/effects/EffLog.java

```diff
@@ -53,11 +53,16 @@
 @Description({"Writes text into a .log file. Skript will write these files to /plugins/Skript/logs.",
 		"NB: Using 'server.log' as the log file will write to the default server log. Omitting the log file altogether will log the message as '[Skript] [&lt;script&gt;.sk] &lt;message&gt;' in the server log."})
 @Examples({"on place of TNT:",
-		"	log \"%player% placed TNT in %world% at %location of block%\" to \"tnt/placement.log\""})
-@Since("2.0")
+	    "	log \"%player% placed TNT in %world% at %location of block%\" to file \"tnt/placement.log\"",
+	    "	log \"%player% placed TNT in %world% at %location of block%\" to file \"tnt/placement.log\"\"with severity of info\"",
+		"	log \"%player% placed TNT in %world% at %location of block%\" to file \"tnt/placement.log\"\"with severity of warning\"",
+        "   log \"%player% placed TNT in %world% at %location of block%\" to file \"tnt/placement.log\"\"with severity of error\""})
+
+
+@Since("2.0, 2.9 (Adds severity to logs)")
 public class EffLog extends Effect {
 	static {
-		Skript.registerEffect(EffLog.class, "log %strings% [(to|in) [file[s]] %-strings%]");
+		Skript.registerEffect(EffLog.class, "log %strings% [(to|in) [file[s]] %-strings%] [with severity (of) (1:warning|2:severe)]");
```

Comment:
I've taken `(of)` as meant to be optional but I also don't know how I feel about it.
```suggestion
		Skript.registerEffect(EffLog.class, "log %strings% [(to|in) [file[s]] %-strings%] [with [the] severity [of] (1:warning|2:severe)]");
```